### DISCUSSION TITLE
Get list of all untranslated pages?

### DISCUSSION THREAD

**Discussion by hpoul** [wagtail/wagtail-localize#522] [Q&A]
Hello,
this is probably more a Wagtail related and not specific to wagtail-localize - but is there a way, after synchronizing pages from one locale to another to show all those "alias" pages which need to be translated in this new locale?
As far as I understand it, once they are translated and the original locale gets updated they will show up in the `Reports -> Translations`? But when they have never been touched, it seems they aren't reported there? 🤔️

thanks,
  Herbert

**Comment by zerolab** [comment]
Something like `Page.objects.filter(alias_of__isnull=False).specific` can give you all alias pages.
The you could probably further filter on specific locales

**Reply by hpoul** [reply]
I actually meant if there is anything in the admin UI i have been missing... Seems likes common use case.. if somebody creates a new page in the source language and publishes it, that it would be visible somewhere that it misses translations...

But i guess I'd have to create a workflow where the author would submit it for translation of something.. anyway, just wanted to make sure I've not missed something obvious, thanks. 😅

**Reply by zerolab** [reply]
Ah, don't think there's anything in the UI yet. Will be happy to review a PR for it ;)
In the meantime you could write your own report and make it suit your particular needs. See https://docs.wagtail.org/en/stable/extending/adding_reports.html#adding-reports for a method
